.github/workflows/build.yml

```diff
@@ -1,41 +1,47 @@
-name: Build
-on: [push, pull_request]
+name: Build Code
+
+on:
+  push:
+  pull_request:
+    types: [opened]
+
+env:
+  INEXOR_VULKAN_VERSION: "1.3.216.0"
+  INEXOR_VULKAN_SDK_PATH: "$GITHUB_WORKSPACE/../vulkan_sdk/"
 
 jobs:
   linux:
     name: ${{ matrix.config.name }}
     runs-on: ubuntu-latest
+    if: ${{ github.event_name == 'pull_request' && github.event.action == 'opened' && !github.event.pull_request.merged || github.event_name == 'push' }}
     container: ubuntu:rolling
     env:
       DEBIAN_FRONTEND: "noninteractive"
-      CONAN_USER_HOME: "${{ github.workspace }}/conan/"
-      CONAN_USER_HOME_SHORT: "${{ github.workspace }}/conan/s"
-      inexor_conan_path: "$HOME/.local/bin"
     strategy:
       fail-fast: false
       matrix:
         config:
           - {
```

Comment:
You can expand the matrix to include the build types (debug and release).

Reply:
Oooh that's nice, ok

Reply:
I improved the config by including the build types. It seems to work, but Ubuntu Clang (Release) seems to fail. Also, it could be that sometimes Windows MSVC builds (both Debug and Release) fail because I upload the entire build folder instead of only restricting the upload to the example binary, will look into that. Sometimes when I re-run the Windows MSVC (Debug&Release), it works in 2nd run. Compilation always works, but uploading build artifacts does not sometimes.

Reply:
Ok, done.